### DISCUSSION TITLE
pdx debugactiveprocess access denied

### DISCUSSION THREAD

**Discussion by apatedolos** [jtpereyda/boofuzz#576] [Q&A] [answered]
Hi

This is a nice fork of the original sulley project!

I am having an issue when trying to run against an FTP application during the debug phase.

`[01:38.36] debugger thread-1498725516 attaching to pid: 196
Exception in thread 1498725516:
Traceback (most recent call last):
  File "C:\Python27\lib\threading.py", line 810, in __bootstrap_inner
    self.run()
  File "process_monitor.py", line 100, in run
    self.dbg.attach(self.pid)
  File "C:\Python27\lib\site-packages\pydbg\pydbg.py", line 226, in attach
    self.debug_active_process(pid)
  File "C:\Python27\lib\site-packages\pydbg\pydbg.py", line 839, in debug_active
_process
    raise pdx("DebugActiveProcess(%d)" % pid, True)
pdx: [5] DebugActiveProcess(196): Access is denied.
`
Can anyone shed any light why this would be happening?

**Comment by apatedolos** [comment, marked as answer]
In case this helps anyone else, this was due to running the application from the user\Desktop.

Moving to C:\\ fixed the access denied on the debuger.

**Comment by jtpereyda** [comment]
This happens when you are trying to attach to a 64-bit app using pydbg which is made for 32-bit applications. Boofuzz doesn't have 64-bit support yet. :'(
